hack/s3_test/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:bullseye-slim
+
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends wget ca-certificates containerd
```

Comment:
Why is containerd needed here?

Reply:
Buildkit does not start without it 
```
WARN[2022-05-06T08:27:58Z] skipping oci worker, as runc does not exist
WARN[2022-05-06T08:27:58Z] skipping containerd worker, as "/run/containerd/containerd.sock" does not exist
buildkitd: no worker found, rebuild the buildkit daemon?
```

May be you have a magical option to add to `buildkitd -debugaddr 0.0.0.0:8060` :)